price-feeder/oracle/provider/gate.go

```diff
@@ -493,6 +493,13 @@ func (p *GateProvider) pongHandler(appData string) error {
 	return nil
 }
 
+// GetAvailablePairs return all available pairs symbol to susbscribe.
+func (p *GateProvider) GetAvailablePairs() (map[string]struct{}, error) {
+	return map[string]struct{}{
+		"UMEEUSDT": {},
+	}, nil
```

Comment:
We should grab this from the API - the gate provider is limited to one exchange rate for candles, but not to this exchange rate specifically

Reply:
done at https://github.com/umee-network/umee/pull/649/commits/3c81e2980ead13bbd940eeafe841fd19e5057827